libsolidity/interface/CompilerStack.cpp

```diff
@@ -97,6 +97,24 @@ void CompilerStack::setEVMVersion(EVMVersion _version)
 	m_evmVersion = _version;
 }
 
+void CompilerStack::setLibraries(std::map<std::string, h160> const& _libraries)
+{
+	solAssert(m_stackState < State::AnalysisSuccessful, "Set libraries after analysis.");
```

Comment:
We explicitly throw CompilerError for other cases of mis-use of the interface.

Reply:
This is a copy of `setEVMVersion` above.